src/commonMain/kotlin/teksturepako/pakku/cli/cmd/Cfg.kt

```diff
@@ -0,0 +1,97 @@
+package teksturepako.pakku.cli.cmd
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.Context
+import com.github.ajalt.clikt.core.PrintHelpMessage
+import com.github.ajalt.clikt.core.terminal
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.multiple
+import com.github.ajalt.clikt.parameters.options.*
+import com.github.ajalt.clikt.parameters.types.boolean
+import com.github.ajalt.clikt.parameters.types.choice
+import kotlinx.coroutines.runBlocking
+import teksturepako.pakku.api.data.ConfigFile
+import teksturepako.pakku.api.data.LockFile
+import teksturepako.pakku.api.projects.ProjectSide
+import teksturepako.pakku.api.projects.UpdateStrategy
+
+class Cfg : CliktCommand()
+{
+    override val printHelpOnEmptyArgs = true
+
+    override fun help(context: Context) = "Configure various options of your modpack or projects"
+
+    // -- PROJECTS --
+
+    private val projectArgs: List<String> by argument("projects").multiple()
+
+    private val sideOpt: String? by option("-s", "--side")
+        .help("Change the side of a project")
+        .choice("client", "server", "both", ignoreCase = true)
```

Comment:
Why can't we use enum for the option here? It's for enum?

Reply:
We can 👍

Reply:
ok. I'll do that after 1.5 hours. Go out for dinner

Reply:
Have a nice dinner

Reply:
BTW. You can push to the branch of my repo directly since it's a PR. If you gonna to do it.